internal/command/artifact/junit.go

```diff
@@ -0,0 +1,265 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+package artifact
```

Comment:
could we rename the package to `junit`? I think artifact is quite generic when it only contains the JUnit implementation? Or maybe `testartifacts`?

Reply:
That's fair- I'm making the name generic too early. I'll rename to be specific to JUnit and if we add more file outputs in future the name can be generalised again

Reply:
Addressed in [023ec87](https://github.com/hashicorp/terraform/pull/36304/commits/023ec87ed3e74f4924c57059e2d37e682447b5ed)